feature/platform/transceiver/tests/zr_firmware_version_test/metadata.textproto

```diff
@@ -0,0 +1,17 @@
+# proto-file: github.com/openconfig/featureprofiles/proto/metadata.proto
+# proto-message: Metadata
+
+uuid: "0f3bb528-28f4-4284-8d2f-de105be09241"
+plan_id: "TRANSCEIVER-3"
+description: "Telemetry: 400ZR Optics firmware version streaming"
+testbed: TESTBED_DUT_400ZR
```

Comment:
Does this testbed guarantee that DUT ports 1 and 2 will be 400ZR?

Reply:
I found the answer; in our case yes.  Our reservation system provides a device containing the the requested transceiver PMD's.